src/pages/dynamic-ntt.module.css

```diff
@@ -0,0 +1,682 @@
+/* Import Poppins font */
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600&display=swap');
```

Comment:
We must never load fronts directly from the google site, as this provides an opportunity for google to track  users.

Either include the font files as assets directly, or add a dependency to `@fontsource/poppins`. 

(For me, this doesn't work anyway as it violates the `style-src` CSP.)